chapters/data-processing.tex

```diff
@@ -0,0 +1,725 @@
+	a two-dimensional dataset formed by rows and columns. 
+	Also called ``rectangular data''.
+	Examples of non-rectangular data are a text, or files, such as images.}
+Similarly, these tables can organize information in multiple ways,
+and not all of them result in easy-to-handle datasets.
+Fortunately, a vast literature of database management has identified the format
+that makes interacting with the data as easy as it can be.
+When applied to statistics, we call data in such format \textbf{tidy}.\sidenote{\textbf{Tidy data:}
+	Datasets where each column represents one variable,
+	each row represents one observation, 
+	and all variables in a table have the same unit of observation.}
+A dataset is tidy when each column represents one variable,\sidenote{\textbf{Variable:} 
+	the collection of all data points that measure the same attribute.}
+each row represents one observation, 
+and all variables in a table have the same unit of observation.
+Every other format is \textit{untidy} or just \textit{messy}.
+This may seem trivial, but raw data, 
+and raw survey data in particular,
+is rarely received in a tidy format.
+To contribute to the confusion, 
+Stata users typically call each column a ``variable'', 
+and each row an ``observation'',
+even though they are not necessarily the same thing.
+
+Every messy dataset is messy in its own way,
+but the most common case of untidy raw data encountered in development research
+are datasets consisting of multiple units of observation stored in the same table. 
+If your rows include multiple nested observational units,
+having an identifying variable for each row doesn't mean having an identifier for each observation.
+Survey data containing nested units of observation is typically 
+imported from survey platforms in \textbf{wide format}.\sidenote{\textbf{Wide data:}
+	a table where a single variable is divided into multiple columns,
+	for example one for each individual.
+}
+Take, for example, a household survey that includes household-level questions,
+as well as a household member roster.
+The resulting raw dataset usually consists of a single table
+where questions from the household member roster are saved in different columns, 
+one for each member, with a corresponding member suffix,
+and household-level questions are represented by one column each.
+So you have one column called \texttt{ownsmotorcycle},
+but a few columns of the format \texttt{gender\_1}, \texttt{gender\_2}, etc.
+The data is saved in this format because it's the most efficient way to transfer it:
+adding different levels of observation into the same table allows for data to be transferred in a single file.
+However, this leads to the widespread practice of interacting with data in wide format for a large part of a project,
+although this is not the most efficient way to do so.
+
+To understand how dealing with wide data can be complicated,
+just suppose you need to calculate the share of women in your sample.
```

Comment:
You jump into this without letting the user know you are still talking about data sets with households as the main unit of observation. It could just as well be read as the individuals is the main unit of observation, and then the point we want to make is lost.

Reply:
addressed in 302cbd5